doc/rst/05_troubleshooting.rst

```diff
@@ -0,0 +1,46 @@
+|newpage|
+
+Troubleshooting
+===============
+
+Common Issues
+-------------
+
+This section provides solutions to common issues that users may encounter when using the library.
+
+#. **Camera Not Detected**: If the camera is not connected or not properly connected, the system will fail to detect it. In these cases, the board will blink red and the following error message will be displayed:
+
+    .. code-block:: console
+
+        TODO
+
+   To resolve this issue, ensure that the camera is properly connected to the board. If the camera is not detected, check the connection and try again.
+
+#. **Invalid Image Configuration**: If the image configuration is invalid, the system will not be able to process the image correctly. This can happen if the image size is not supported by the camera or if the image format is not supported. In this case, the following error message will be displayed:
+
+    .. code-block:: console
+
+        xrun: Program received signal ET_ECALL, Application exception.
+        0x00081a9a in camera_isp_coordinates_compute (img_cfg=0xd2280)
+
+   To resolve this issue, ensure that the image configuration is valid and supported by the camera. Check the camera documentation for supported image sizes and formats.
+
+#. **Missing Timings**: It may occur that the camera is not able to provide the timings for a specific mode. This can be due to a variety of reasons, such as the camera has no time to process each line before the next one arrives, the camera is configured faster than it can handle, either the PHY or the SHIM. If this happens, the camera will not be able to provide the timings and the ISP will receive invalid packets. This will result in an error message similar to the following:
```

Comment:
```suggestion
#. **Missing Timings**: It may occur that the camera is not able to meet the timings for a specific mode. This can be due to a variety of reasons, such as the camera running out of time while processing the current line before the next one arrives, the camera is configured faster than it can handle, either the PHY or the SHIM. If this happens, the camera will not be able to meet the required timing and the ISP will receive invalid packets. This will result in an error message similar to the following:
```